src/test/kotlin/racingcar/CarTest.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.Test
+import racingcar.model.Car
+
+class CarTest {
+
+    @Test
+    fun `자동차 이름이 5글자 이상인 경우`() {
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/91#discussion_r464049852

```suggestion
    fun `자동차 이름이 5글자 초과인 경우`() {
```

Reply:
수정하였습니다.